scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -536,12 +540,17 @@ task IndelsVariantRecalibrator {
 
     Int disk_size
     String gatk_docker = "us.gcr.io/broad-gatk/gatk:4.1.9.0"
+    Int? machine_mem_gb
   }
 
+  Int machine_mem = select_first([machine_mem_gb, 30])
+  Int java_mem = machine_mem - 5
+
+
   command <<<
     set -euo pipefail
 
-    gatk --java-options -Xms24g \
+    gatk --java-options -Xmx~{java_mem}g \
```

Comment:
does this need to be -Xms like it was before?

Reply:
that's a great question! I just ended up copying the -Xm param from the other tasks which are -Xmx. but I think they should all be -Xms. any other opinions before I change this back and change the other -Xmx